docs/semgrep-ci/configuration-reference.md

```diff
@@ -19,6 +19,54 @@ While environment variables are the preferred way to configure Semgrep CI, pass
 SEMGREP_RULES="p/security-audit p/secrets"
 ```
 
+## Suppressing blocking comments or errors
+
+Semgrep in CI can block a pull request (PR) or a merge request (MR) from being merged. This block can be caused by Semgrep's blocking comment or Semgrep's in CI internal error. One option to avoid blocked PRs and MRs is to change configuration of blocking comments in the rule board of Semgrep App. Another option that Semgrep in CI allows you is to disable blocking comments without changing the setup of rules in the rule board.
+
+Configure Semgrep in CI behavior to suppress blocking comments or internal Semgrep in CI errors blocking your pipeline by the following options in your Semgrep in CI YAML configuration file: 
+
+- `semgrep ci` - Semgrep in CI fails on blocking comments, CI fails on internal error.
+- `semgrep ci || [ $? != 1 ]` - Semgrep in CI fails on blocking comments, CI ignores internal error.
+- `semgrep ci || true` - Semgrep in CI passes on blocking findings, CI ignores internal error.
```

Comment:
@underyx should I change both "ignores" (lines 29 and 30) back to "passes"? I guess ignore sounds more human...

Reply:
Done!